sktime/forecasting/model_selection/tests/test_split.py

```diff
@@ -462,7 +462,13 @@ def test_split_by_fh(index_type, fh_type, is_relative, values):
         # )
     y = _make_series(20, index_type=index_type)
     cutoff = y.index[10]
-    fh = _make_fh(cutoff, values, fh_type, is_relative)
+    fh = _make_fh(
+        cutoff=cutoff,
+        steps=values,
+        fh_type=fh_type,
+        is_relative=is_relative,
+        freq=infer_freq(y),
```

Comment:
I think we want this test to be "stronger", i.e., to work without `freq` pased to `_make_fh`